plugin/src/sidebar.ts

```diff
@@ -55,6 +56,8 @@ const sidebarInstance: PluginExtensionInstance = reearth.plugins.instances.find(
 
 // ************************************************
 // initializations
+let buildingSearchIsOpen = false;
```

Comment:
Can you move this up where the other xxIsOpen = false let statements?

Reply:
Sure. Updated.